README.md

```diff
@@ -317,6 +317,80 @@ Example for setting a minimum bid value of 0.06 ETH:
 
 Optionally, the `-metrics` flag can be provided to expose a prometheus metrics server. The metrics server address/port can be changed with the `-metrics-addr` (e.g., `-metrics-addr localhost:9009`) flag.
 
+### Enabling timing games 
+The **Timing Games** feature allows `mev-boost` to optimize block proposal by strategically timing `getHeader` requests to relays. Instead of sending a single request immediately, it can delay the initial request and send multiple follow-up requests to capture the latest, most valuable bids right before the proposal deadline.
+
+To enable timing games, you must provide a YAML configuration file using the `-config` flag:
+
+```bash
+./mev-boost -config config.yaml
+```
+**Notice:** This feature is strictly meant for advanced users and extra care should be taken when setting up timing game associated parameters
+
+#### 1. Global Timeouts
+These settings apply to all relays and define the hard boundaries for the `getHeader` operation.
+
+*   **`timeout_get_header_ms`** (optional, default: 900ms)
```

Comment:
How did we choose defaults 900 and 1000? And could we suggest a general practice to the users for picking different numbers?